src/templates/packages.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Helmet from "react-helmet"
+
+class PackageTemplate extends React.Component {
+  constructor() {
```

Comment:
if we don't have any custom code to put in `constructor`, we can remove the whole method.

also another tip with React, if you don't need to work with any lifecycle hooks, you can write your component as a function directory:

```js
function PackageTemplate() {
  return (
    <div>
      ...
    </div>
  );
}
```